frontend/sass/norent/styles.scss

```diff
@@ -174,11 +178,11 @@ $jf-logo-height: 70px;
     }
   }
   @include tablet {
-    height: calc(#{$jf-logo-height} + 50px);
+    height: calc(#{$jf-logo-height} + 20px);
     figure.image img {
       @include is-horizontal-center;
-      height: calc(#{$jf-logo-height} + 50px);
-      width: calc(3 * (#{$jf-logo-height} + 50px));
+      height: calc(#{$jf-logo-height} + 20px);
+      width: calc(3 * (#{$jf-logo-height} + 20px));
```

Comment:
Should the `20px`  be moved to its own variable/constant?  Could be useful for readability if it represents some kind of meaningful value, but not a problem if it's arbitrary.

Reply:
oh yeah I went back and forth about this— it's just an offset to make the logo a bit bigger on tablet/desktop. I guess it's good to keep it as a variable though...